cclib/bridge/__init__.py

```diff
@@ -27,4 +27,6 @@
 if find_package("iodata"):
     from cclib.bridge.cclib2horton import makehorton
 
+if find_package("pyscf"):
+    from cclib.bridge.cclib2pyscf import makepyscf, makepyscf_mos
```

Comment:
Do you think we will need to continue to support two version, or do you see these merging in the future?

Reply:
This is a good questions. Since pyscf stores the molecule object separate from the mo_coeff object, I thought this might be a cleaner solution in cases where one may only want a molecule object. The alternative could be the makepyscf function returning either one or two things. The molecule object could be returned in the case that mocoeffs are not present in the cclib data. Otherwise the function could return both the molecule object and the mo_coeff object.

Is there a way you would prefer or maybe another way I haven't considered?

Reply:
I think this is fine, since I operate under the assumption that PySCF is a rapidly moving target...